Vizzuality/heco-invest
# Let 905 fe implement the open calls list in the dashboard of investors

This PR adds the open calls list on the Investor dashboard

## Description
Investor users are able to see their list of open calls, search it and sort it. They can also quickly go to the public page of a launched or closed open call.

Only the “Go to open call page” action should be implemented in the “3 dot” menu because all of its other actions are split to other tasks.

Figma: [HeCo - Work in progress](https://www.figma.com/file/KC49uMR9t1YARmFR0Sr6aN/HeCo---Work-in-progress?node-id=2%3A5403)

## Testing instructions

1. Sign in as an approved investor
2. Go toi dashboard
3. You will be redirected to dashboard/open-calls. You should see a table with the account open calls.
4. You should be able to filter by any column value except 'value'
5. You should be able to sort all columns except 'instrument type' and 'actions' (the last one)

## Tracking

[905](https://vizzuality.atlassian.net/browse/LET-905)
